Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -0,0 +1,227 @@
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: model.graphicsOverlays)
+            .onSingleTapGesture { _, point in
+                switch selectedGraphicType {
+                case .facilities:
+                    model.addFacilityGraphic(at: point)
+                case .barriers:
+                    model.addBarrierGraphic(at: point)
+                }
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Picker("Mode", selection: $selectedGraphicType) {
+                        ForEach(GraphicType.allCases, id: \.self) {
+                            Text($0.label)
+                        }
+                    }
+                    .pickerStyle(.segmented)
+                    Spacer()
+                    Menu {
+                        Stepper("Second: \(secondTimeBreak)", value: $secondTimeBreak, in: 1...15)
+                        Stepper("First: \(firstTimeBreak)", value: $firstTimeBreak, in: 1...15)
+                    } label: {
+                        Label("Time", systemImage: "gear")
+                    }
+                    Spacer()
+                    Button("Service Area") {
+                        Task {
+                            do {
+                                try await model.showServiceArea(timeBreaks: [Double(firstTimeBreak), Double(secondTimeBreak)])
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    }
+                    Spacer()
+                    Button("Clear", systemImage: "trash.fill") {
+                        model.removeAllGraphics()
+                    }
+                }
+            }
+            .errorAlert(presentingError: $error)
+    }
+}
+
+private extension ShowServiceAreaView {
+    // MARK: - GraphicType
+    
+    enum GraphicType: Equatable, CaseIterable {
+        case facilities, barriers
+        
+        /// The string representation of this graphic type.
+        var label: String {
+            switch self {
+            case .barriers: "Barriers"
+            case .facilities: "Facilities"
+            }
+        }
+    }
+}
+
+private extension ShowServiceAreaView {
+    // MARK: - Model
+    
+    class Model: ObservableObject {
+        /// A map with terrain style centered over San Diego.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTerrain)
+            map.initialViewpoint = Viewpoint(
+                center: Point(
+                    x: -13041154,
+                    y: 3858170,
+                    spatialReference: .webMercator
+                ),
+                scale: 60_000
+            )
+            return map
+        }()
+        
+        let facilitiesGraphicsOverlay: GraphicsOverlay = {
```

Comment:
Remember to make these overlay properties private since they aren't used outside of this type.